base/markdown/render/plain.jl

```diff
@@ -38,6 +38,14 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, q::BlockQuote)
+    s = sprint(buf -> plain(buf, q.content))
```

Comment:
If it can be done in a single pass without becoming overly complex then that would be great.